rtos/Thread.h

```diff
@@ -246,7 +246,7 @@ class Thread : private mbed::NonCopyable<Thread> {
 
     /** Set the specified Thread Flags for the thread.
       @param   signals  specifies the signal flags of the thread that should be set.
-      @return  previous signal flags of the specified thread or osFlagsError in case of incorrect parameters.
+      @return  resultant signal flags of the specified thread or osFlagsError in case of incorrect parameters.
     */
     int32_t signal_set(int32_t signals);
```

Comment:
can we word it more like that http://www.keil.com/pack/doc/CMSIS/RTOS2/html/group__CMSIS__RTOS__ThreadFlagsMgmt.html#ga6f89ef9caded1d9963c7b12b0f6412c9

Reply:
done